### PR TITLE
[Merged by Bors] - chore: update fluvio to 0.10.9

### PR DESCRIPTION
**Pull request by nacardin**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -21,8 +21,8 @@ rumqttc = { version = "0.20", features = ["url", "websocket"] }
 rustls = { version = "0.20" }
 rustls-native-certs = { version = "0.6" }
 
-fluvio = { git = "https://github.com/infinyon/fluvio", tag = "v0.10.8-dev-3" }
-fluvio-connector-common = { git = "https://github.com/infinyon/fluvio", tag = "v0.10.8-dev-3", features = ["derive"]}
+fluvio = { git = "https://github.com/infinyon/fluvio", tag = "v0.10.9-dev-1" }
+fluvio-connector-common = { git = "https://github.com/infinyon/fluvio", tag = "v0.10.9-dev-1", features = ["derive"]}
```
Comment (morenol) [inline Cargo.toml:25]: update to release version

File: hub/package-meta.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
```
Comment (morenol) [inline hub/package-meta.yaml:1]: we can delete this file. This is autogenerated:

**Review by morenol — APPROVED** [review]
lgtm, but please update versions in the readme

File: hub/package-meta.yaml (hunk 2 above)
Comment (nacardin) [reply hub/package-meta.yaml]: done

**Review by morenol — APPROVED** [review]

**Pull request closed without merging**